native/src/test/java/io/ballerina/stdlib/grpc/logging/GrpcLogManagerTest.java

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.grpc.logging;
+
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.stdlib.grpc.logging.formatters.GrpcAccessLogFormatter;
+import io.ballerina.stdlib.grpc.logging.formatters.GrpcTraceLogFormatter;
+import io.ballerina.stdlib.grpc.logging.formatters.JsonLogFormatter;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.SocketHandler;
+
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_LOG_CONSOLE;
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_LOG_FILE_PATH;
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_TRACE_LOG_HOST;
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_TRACE_LOG_PORT;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * A unit test class for Grpc module GrpcLogManager class functions.
```

Comment:
```suggestion
 * A unit test class for gRPC module GrpcLogManager class functions.
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/413/commits/303299de9612deb38959f15b7389ed952825454a